src/data/base/aerodromeLpPools.json

```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "name": "aerodrome-usdc-usd+",
+    "address": "0x418457Ca08fA5EC77f811B105F2c585cd051Ac10",
```

Comment:
This is in the wrong file? This is stable pool and should be in aerodromeStableLpPools?

Reply:
fixed it all I hope. Thanks for your patience, shouldn't happen again as I can let the script do it now and even found out what "run the api" means!